pegasus/sites.v3/code.org/views/music/music_lab_featured_artists.haml

```diff
@@ -0,0 +1,10 @@
+- artist_index = 5
+
+- artist_photo = ["/images/music-lab/chainsmokers.png", "/images/music-lab/rosa-linn.png", "/images/music-lab/carly-rae-jepson.png", "/images/music-lab/dove-cameron.png", "/images/music-lab/aloe-blacc.png"]
```

Comment:
Still waiting on the CRJ photo, but can add it once we get it, or remove it before launch if needed.